### ISSUE TITLE
Warnings in the Browser.sandbox (etc) snippets

### ISSUE THREAD

**Issue by lydell** [elm-tooling/elm-language-server#555]
This issue is about these snippets:

- `Browser.sandbox`
- `Browser.element`
- `Browser.document`
- `Browser.application`

Expanding them gives these warnings:

- Unused top level definition `main` (ElmLS)
- Using a record is obsolete if you only plan to store a single field in it. (ElmLS)
- Unused pattern variable `model` (ElmLS)

Solutions:

- Expose `main`. This is especially confusing and non-obvious for newcomers.
- Have two fields by default? Ignore the rule for `Model`? Remove the rule altogether? Using a record for `Model` is the best way to get started – we should not nudge newcomers to do other things.
- `model` being unused is probably fine – it’s nudging the user to use it in the `view`.

Also:

- `init` uses `Model ""` instead of `{ property = "" }`. The latter syntax is generally preferred, and helps newcomers?
- Some of the snippets use `property : propertyType` which causes a somewhat confusing error message. Better to use `property : String` or something?
- `Browser.element` contains `key : Nav.Key, url : Url.Url` but those don’t make sense? (Also, `Nav` and `Url` aren’t imported.)

In summary, I’d love if the snippets:

- Compiled by default, as much as possible
- Don’t have warnings, as much as possible
- Are elm-format:ed
- Nudge newcomers in the right direction
